Sources/Mapper.swift

```diff
@@ -433,7 +433,7 @@ extension Mapper where N: Hashable {
 		#if swift(>=4.1)
 		return Set(JSONArray.compactMap(map))
 		#else
-		return Set(JSONArray.flatMap(map))
+		return Set(JSONArray.compactMap(map))
```

Comment:
**Same comment as above.**
Please just remove `else` statement if it is necessary because now `if` and `else` blocks has same actions.